website/docs/contributing/general/ADR/domain-language.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "ADR: Domain language"
+---
+
+## Background
+
+In the codebase, we have seen the need to define a domain language that we use to refer to features, methods to keep it consistent across the codebase. This ADR will contain a growing list of domain language used to keep the consistency across the codebase.
+
+## Decision
+
+We have decided to use the same domain language for the features we develop. Each feature will have it's own domain language to keep it consistent across the codebase.
+
+## Change requests domain language
+
+* *Change request*
+
+An entity referring to the overarching data structure of a change request. A change request contains changes, and can be approved or rejected.
+
+* *Changes*
+
+An entity referring to a single change within a change request
+
+* *Discard*
+
+A naming convention used for deleting a single change of a change request, or discarding an entire change request
+
+* *Pending*
+
+A naming convention used to describe a change requests state. A pending change request is in one of these three states: `Draft | In review | Approved`
```

Comment:
Apparently there are 3 well defined underlying states, that we're "masking" under simpler terms for the end user. If that's not the case, please ignore the rest of this comment.
If each sub-state falls under the bigger umbrella of "Pending", I'd appreciate a clarification for the end user point of view. Examples:
- a Draft change is _Pending to be assigned a Reviewer_ 
- an In Review change is _Pending to be Approved_
- an Approved change is _Pending to be applied_

Reply:
We don't use this term in the user facing code, but rather as our own term for naming the grouping of these states. For the user, they will only relate to the state themselves as modeled: `Draft | In review | Approved | Applied | Cancelled`